modules/hostap/CMakeLists.txt

```diff
@@ -17,499 +17,499 @@ set(CMAKE_EXE_LINKER_FLAGS "--specs=nosys.specs -lnosys")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DMISSING_SYSCALL_NAMES")
```

Comment:
Personally I find it easier to read the CMake file with the current indentation. Making indentation 2 spaces long, at least for me the code looks now quite "crowded".

Reply:
My comment is not a deal breaker, I can certainly live with 2 space indentation in this file if we really want to do this.

Reply:
is there a coding-rule about this?
I always thought zephyr required 8-spaces tab = =.

Reply:
Is there specific reason to use spaces (2) as the indentation?

Reply:
That's what we use in downstream fork (NCS), https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style says just to use spaces. @nordicjm  any ideas why we adopted 2spaces for Cmake?